src/home-page/NoMatch404.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import { MainContainer } from '../layout/components/MainContainer';
+
+const NoMatchTypography = styled('div')`
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction:column;
+  margin: auto;
+  min-height: 70vh;
+`;
+
+const HeaderTypography = styled(Typography)`
+  font-size:10em;
+`;
+const SubtitleTypography = styled(Typography)`
+  font-size:2em;
+  margin: ${props => props.theme.spacing(2)}px;  
+`;
+
+const StyledButton = styled(Button)`
+color: white;
```

Comment:
Maybe we can use here our them variables as well (I mean add white to our them and use it here instead hard code white here). It will be easier in the future to change colors in one place then looking for them in different components. :)

Reply:
done